src/reducers/versions.tsx

```diff
@@ -756,7 +759,13 @@ const reducer: Reducer<VersionsState, ActionType<typeof actions>> = (
     case getType(actions.loadDiff): {
       const { baseVersionId, headVersionId, version } = action.payload;
 
-      const { entries, selectedPath } = getVersionInfo(state, headVersionId);
+      const headVersion = getVersionInfo(state, headVersionId);
+      if (!headVersion) {
+        _log.error(`Version missing for headVersionId: ${headVersionId}`);
```

Comment:
I'll be honest. I don't think it's important to use dependency injection to test a log message 😄 If you add a test that runs this line of code to make sure no errors are thrown, that's enough. TypeScript will catch all other mistakes that could go wrong with this log message. It was way more important to fully test function calls like this before static analysis.

Reply:
Yep, this is more to reach 100% code coverage than anything else now.